src/Build/Evaluation/Expander.cs

```diff
@@ -4251,6 +4251,11 @@ private static bool TryGetArgs(object[] args, out int arg0, out string arg1)
                 }
 
                 arg1 = args[1] as string;
+                if (arg1 == null && args[1] is char ch)
```

Comment:
not affecting this PR: At some point somebody has to extract all this C# interpretation code outside of Expander.cs and into its own class :)

Reply:
At some point somebody should rewrite the Evaluator and kill this class altogether ;)

Reply:
They mythical "some point" when everything is made nice :)